packages/jest-changed-files/src/git.ts

```diff
@@ -72,12 +74,12 @@ const adapter: SCMAdapter = {
   },
 
   getRoot: async cwd => {
-    const options = ['rev-parse', '--show-toplevel'];
+    const options = ['rev-parse', '--show-cdup'];
 
     try {
       const result = await execa('git', options, {cwd});
 
-      return result.stdout;
+      return path.join(cwd, result.stdout);
```

Comment:
```suggestion
      return path.resolve(cwd, result.stdout);
```

Reply:
In this case it makes no difference, because `cwd` is always absolute, and `result.stdout` is always relative. But ok, i guess `resolve` is safer in some pessimistic cases.